srcpkgs/R-cran-tibble/template

```diff
@@ -1,19 +1,19 @@
 # Template file for 'R-cran-tibble'
 pkgname=R-cran-tibble
-version=3.0.1
+version=3.0.3
 revision=1
 build_style=R-cran
 makedepends="R-cran-cli R-cran-crayon R-cran-ellipsis
  R-cran-fansi R-cran-lifecycle R-cran-magrittr
  R-cran-pillar R-cran-rlang R-cran-pkgconfig R-cran-vctrs"
-depends="R-cran-cli R-cran-crayon R-cran-ellipsis
- R-cran-fansi R-cran-lifecycle R-cran-magrittr
- R-cran-pillar>=1.4.3 R-cran-rlang R-cran-pkgconfig R-cran-vctrs"
+depends="R-cran-cli R-cran-crayon>=1.3.4 R-cran-ellipsis>=0.2.0
```

Comment:
Unless you know way to fill template with required versions automatically, I would prefer to assume system is updated as whole.

Reply:
Sure no problem, just to clarify you mean just leave off the version numbers and assume we will always be using the latest release?

Reply:
yes

Reply:
Done, I'll keep that in mind for future PRs.